java-racingcar/src/test/java/com/game/javaracingcar/level1/SetTest.java

```diff
@@ -0,0 +1,48 @@
+package com.game.javaracingcar.level1;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.apache.logging.log4j.util.Strings;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("요구사항 1")
+    @Test
+    void Requirements01 () {
+        assertThat(numbers.size()).isEqualTo(3);
```

Comment:
다음과 같이 작성할수도 있을 것 같습니다.
```suggestion
        assertThat(numbers).hasSize(3);
```